vmware_aria_operations_integration_sdk/project.py

```diff
@@ -107,27 +139,76 @@ def name(self) -> str:
         return get_project_name(self.path)
 
     def record(self) -> None:
-        config_file = os.path.join(self.path, "config.json")
+        config_file = os.path.join(self.path, CONFIG_FILE_NAME)
+        connections_file = os.path.join(self.path, CONNECTIONS_FILE_NAME)
         set_config_value(
-            "connections", [conn.__dict__ for conn in self.connections], config_file
+            CONNECTIONS_CONFIG_CONNECTIONS_LIST_KEY,
+            [conn.__dict__ for conn in self.connections],
+            connections_file,
         )
-        set_config_value("docker_port", self.docker_port, config_file)
+        set_config_value(CONFIG_DOCKER_PORT_KEY, self.docker_port, config_file)
 
     @classmethod
     def extract(cls, path: str) -> Project:
-        local_config_file = os.path.join(path, "config.json")
+        local_config_file = os.path.join(path, CONFIG_FILE_NAME)
+        connections_file = os.path.join(path, CONNECTIONS_FILE_NAME)
+
         if not os.path.isfile(local_config_file):
-            with open(local_config_file, "w") as config:
-                json.dump({}, config, indent=4, sort_keys=True)
+            with open(local_config_file, "w") as _config:
+                json.dump({}, _config, indent=4, sort_keys=True)
+
+        _safe_append_to_gitignore(
+            os.path.join(path, ".gitignore"), CONNECTIONS_FILE_NAME
+        )
+
+        connections_data = {}
+        connection_file_keys = [
+            CONNECTIONS_CONFIG_SUITE_API_HOSTNAME_KEY,
+            CONNECTIONS_CONFIG_SUITE_API_USERNAME_KEY,
+            CONNECTIONS_CONFIG_SUITE_API_PASSWORD_KEY,
+            CONNECTIONS_CONFIG_SUITE_API_CONNECTION_KEY,
+            CONNECTIONS_CONFIG_CONNECTIONS_LIST_KEY,
+        ]
+
+        with open(local_config_file, "r+") as _config:
+            json_config = json.load(_config)
+            docker_port = json_config.get(CONFIG_DOCKER_PORT_KEY, 8080)
+
+            for key in connection_file_keys:
+                if key in json_config:
+                    logger.warning(
+                        f"{CONNECTIONS_FILE_NAME} key '{key}' found in {CONFIG_FILE_NAME}."
+                    )
+                    connections_data[key] = json_config.get(key)
+                    logger.debug(f"Deleting {key} from {CONNECTIONS_FILE_NAME}")
+                    del json_config[key]
+
+            if len(connections_data):
+                _config.seek(0)
+                logger.info(
+                    f"deleting {CONNECTIONS_FILE_NAME} keys from {CONFIG_FILE_NAME}"
+                )
+                json.dump(json_config, _config, indent=4, sort_keys=True)
+                _config.truncate()
+                logger.info(
+                    f"To learn more about '{CONNECTIONS_FILE_NAME}' file visit TODO: ADD LINK HERE"
```

Comment:
Put a message like this in the prompt, so users can find more info before they commit to migration.